src/components/CreditCard/CreditCard.styles.ts

```diff
@@ -31,23 +31,24 @@ export const DivStyledCreditCard = styled.div<TStyleProps>`
     height: 154px;
     width: 277px;
 
-    label {
-        height: 10px;
-    }
-
     ${({ isExpired, pressed }) =>
         isExpired ? expiredStyles : pressed ? pressedStyles : ''}
 `;
 
 export const DivStyledFlex = styled.div`
-    display: flex;
-    justify-content: space-between;
     align-items: center;
+    display: flex;
+    justify-content: flex-end;
```

Comment:
This style effects the creditcards footer and therefore the style does not match anymore

Should look like:
![creditCardFigma](https://user-images.githubusercontent.com/89942527/157418877-3feaeda2-6822-469c-aa63-82bcc1eae2c6.png)

But looks like:
![looksLike](https://user-images.githubusercontent.com/89942527/157419359-66162c25-e9c0-4c1b-a83d-433e2f7f0f8e.png)

Reply:
ok nice, will fix now